fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java

```diff
@@ -244,12 +244,6 @@ public GroupId addTableToGroup(long dbId, OlapTable tbl, String groupName, Group
                 group2Schema.put(groupId, groupSchema);
             }
 
-            if (groupAlreadyExist) {
```

Comment:
do we still need this check? assume we have a colocation group `groupa` which includes only olap tables, add we have a lake table `colocate_with='groupa'`, what will happen in this situation? is `assignedGroupId` null or not?

Reply:
this will not happen since we have run_mode in fe